src/Illuminate/Queue/RedisQueue.php

```diff
@@ -129,8 +129,9 @@ public function pop($queue = null)
 
         $queue = $this->getQueue($queue);
 
+        $this->migrateExpiredJobs($queue.':delayed', $queue);
         if (! is_null($this->expire)) {
```

Comment:
Would probably add a newline between the $this->migrateExpiredJobs(... and the if condition no?